src/index.js

```diff
@@ -4,6 +4,15 @@ import { saveTasks, savedTasks } from './modules/edit.js';
 const taskInput = document.getElementById('input');
 const addButton = document.getElementById('button');
 const todoList = document.getElementById('todo-list');
+const clearButton = document.getElementById('clearButton');
+
+// Load tasks from local storage
+function loadTasks() {
+  const tasks = localStorage.getItem('tasks');
+  if (tasks) {
+    savedTasks.push(...JSON.parse(tasks));
+  }
+}
```

Comment:
- [ ] Upon reloading the page, the list of todo list is just multiplying please make sure your list is only populated with the number of tasks that were already present in the list once

https://github.com/SAZZAD1Q2/New-TO-Do-List/assets/44798044/0688cbd4-1ecd-415a-8756-c722225b1d8b